src/client/src/tscSubquery.c

```diff
@@ -101,13 +101,12 @@ static bool allSubqueryDone(SSqlObj *pParentSql) {
 
 bool subAndCheckDone(SSqlObj *pSql, SSqlObj *pParentSql, int idx) {
   SSubqueryState *subState = &pParentSql->subState;
-  assert(idx < subState->numOfSub);
```

Comment:
It doesn't change the behavior of code on normal conditions, but just ignores the case of index being out of range on concurrent destruction of subqueries, since concurrent accessing to statuses of subqueries is not rare.